### ISSUE TITLE
Logits and Softmax

### ISSUE THREAD

**Issue by LazerLambda** [LazerLambda/Promptzl#26]
- logits for `forward`
- softmax for `classify`
- Adapt tests
